### PR TITLE
Adds statistics to requirements

### PR DESCRIPTION
**Pull request by halfak**
Added 'statistics' to requirements for compatibility with python 3.x where x < 4

### CODE REVIEW

**Comment by halfak** [comment]
FYI, I checked and this seems to cause no harm for python 3.4 installs.

**Comment by he7d3r** [comment]
`$ ack-grep revscores`
still returns a few results (mostly in examples).

File: setup.py [hunk 1]
```diff
@@ -8,11 +8,11 @@ def read(fname):
...
 
 def requirements(fname):
-    return [line.strip() 
+    return [line.strip()
             for line in open(os.path.join(os.path.dirname(__file__), fname))]
 
 setup(
-    name="revscores",
+    name="revscoring",
     version=read('VERSION').strip(),
```
Comment (he7d3r) [inline setup.py]: Consider increasing the version number from 0.0.2.

Comment (halfak) [inline setup.py]: The only time we should be increasing version numbers is when we publish a release ( i.e. upload a snapshot to PyPI)

**Pull request merged**